hive-metastore/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -58,9 +58,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class HiveCatalog extends BaseMetastoreCatalog implements Closeable, SupportsNamespaces {
+public class HiveCatalog extends BaseMetastoreCatalog implements AutoCloseable, SupportsNamespaces {
```

Comment:
It doesn't look like these changes are needed. Could we move the changes in this file to a separate PR?

Reply:
I'm not sure this diff "technically" change anything. Sounds unnecessary unless explained.

Reply:
I was under the impression that we couldn't use `with { } `. But this seems incorrect. I'll revert this change. Thanks for pointing it out, so we can keep the history clean 👍